### PR TITLE
CELDEV-952 - drop IE < 9 support and correct transform-scale

### PR DESCRIPTION
**Pull request by fpichler**
https://synjira.atlassian.net/browse/CELDEV-952

### CODE REVIEW

File: src/main/webapp/resources/celJS/celpresentation/CelementsSlideShow.js [hunk 1]
```diff
@@ -335,216 +325,216 @@ window.CELEMENTS.presentation.SlideShow = function(containerId) {
...
-        var _me = this;
-        callbackFN = callbackFN || function(newSlideContent) {};
-        var params = {
-          'xpage' : 'celements_ajax',
-          'ajax_mode' : 'pageTypeWithLayout',
-          'ajax' : '1'
+      _preloadSlide: function (slideFN, callbackFN) {
+        const _me = this;
+        callbackFN = callbackFN || function (newSlideContent) { };
+        let params = {
```
Comment (msladek) [inline src/main/webapp/resources/celJS/celpresentation/CelementsSlideShow.js:534]: `params` is never reassigned
```suggestion
        const params = {
```

File: src/main/webapp/resources/celJS/celpresentation/CelementsSlideShow.js [hunk 2]
```diff
@@ -639,75 +629,73 @@ window.CELEMENTS.presentation.SlideShow = function(containerId) {
...
-//            parentWidth, parentHeight);
+        //ensure that the width is not influenced by the parents by setting position to absolute
+        slideRoot.setStyle({
+          'position': 'absolute'
+        });
+        //use scrollWidth and scrollHeight, because it works correctly inside iframes.
+        const slideOuterHeight = slideRoot.scrollHeight;
+        const slideOuterWidth = slideRoot.scrollWidth;
+        if (_me._debug) {
+          console.log('_centerCurrentSlide slideRoot dim: ', _me._htmlContainerId, slideOuterWidth,
```
Comment (msladek) [inline src/main/webapp/resources/celJS/celpresentation/CelementsSlideShow.js]:
```suggestion
          console.debug('_centerCurrentSlide slideRoot dim: ', _me._htmlContainerId, slideOuterWidth,
```
and all following occurences of `console.log` within debug check

File: src/main/webapp/resources/celJS/mobile/MobileSupport.js [hunk 3]
```diff
@@ -118,13 +118,10 @@ CELEMENTS.mobile.Dimensions.prototype = {
...
-    getZoomStyles : function(zoomFactor, fullWidth, fullHeight) {
-      var _me = this;
-      var zoomStyles = {};
-      //zoom for IE < 9 else transform:scale
-      if (_me.isMobile.IE() && (_me.version() < 9)) {
-        zoomStyles['zoom'] = zoomFactor;
-      } else if (_me.isMobile.IE() && (_me.version() < 10)) {
+    getZoomStyles: function (zoomFactor, fullWidth, fullHeight) {
+      const _me = this;
+      let zoomStyles = {};
```
Comment (msladek) [inline src/main/webapp/resources/celJS/mobile/MobileSupport.js]:
```suggestion
      const zoomStyles = {};
```

**Review by msladek — CHANGES_REQUESTED** [review]

File: src/main/webapp/resources/celJS/celpresentation/CelementsSlideShow.js (hunk 1 above)
Comment (fpichler) [reply src/main/webapp/resources/celJS/celpresentation/CelementsSlideShow.js:534]: Not true. See line 549

**Review by msladek — APPROVED** [review]

**Pull request merged**